tests/pipelines/test_execution_utils.py

```diff
@@ -104,6 +104,34 @@ def clean_test_pipeline(enter_test_pipeline_directory):  # pylint: disable=unuse
         Pipeline(profile="local").clean()
 
 
+def test_create_required_step_files(tmp_path):
+    class TestStep(BaseStepImplemented):
+        def __init__(self):  # pylint: disable=super-init-not-called
+            pass
+
+        @property
+        def name(self):
+            return "test_step"
+
+    def check_required_files(check_exist):
+        for required_file in [
+            "steps",
+            "steps/ingest.py",
+            "steps/split.py",
+            "steps/train.py",
+            "steps/transform.py",
+            "steps/custom_metrics.py",
+        ]:
+            assert not check_exist ^ (tmp_path / required_file).exists()
```

Comment:
```suggestion
            assert (tmp_path / required_file).exists() is check_exist
```

This might be easier to understand.